docs/research/Translation Models and S_M4T/Readme.md

```diff
@@ -2,4 +2,13 @@
 
 **As seen in the [POC](../../../example/POC_Seamless_M4T/)**
 
-# TODO
\ No newline at end of file
+## Parallel models
```

Comment:
Please fill this out properly describing your process, what you have tried and what did/didnt work

Reply:
Maybe consider taking some of the poc readme content here. This is the place where knowledge is stored not in some poc directory nobody will ever open again